ui/src/app/pages/access/controlled-tier-card.tsx

```diff
@@ -32,7 +32,7 @@ import { ModuleIcon } from './module-icon';
 import { ModulesForAnnualRenewal } from './modules-for-annual-renewal';
 import { ModulesForInitialRegistration } from './modules-for-initial-registration';
 
-const handleClickSupportButton = (url) => () => {
```

Comment:
Changed this name because there are now 2 "click support" actions.  This one is specifically to Request Access, and the action actually taken is not always to email support.  Sometimes it is to visit a custom URL.

Actually mailing support **is** the desired action for the other support click (new for this PR)